BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/UIView+RoundCorners.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  UIView+RoundCorners.swift
+//  
+//
+//  Created by Valentina Iancu on 30.06.23.
+//
+
+import UIKit
+
+extension UIView {
+	func round(corners: UIRectCorner = [UIRectCorner.allCorners], radius: CGFloat) {
+		layer.maskedCorners = CACornerMask(rawValue: corners.rawValue)
+		layer.cornerRadius = radius
+		layer.masksToBounds = radius != 0
```

Comment:
Clarification: how should we read this?
		`layer.masksToBounds = radius != 0`
And why we don't need `!=0` for
		`layer.cornerRadius = radius`